utils/nativefuncs.go

```diff
@@ -165,13 +165,31 @@ func RegisterNativeFuncs(vm *jsonnet.VM, resolver Resolver) {
 
 	vm.NativeFunction(&jsonnet.NativeFunction{
 		Name:   "parseHelmChart",
-		Params: []jsonnetAst.Identifier{"releaseName", "namespace", "chartData", "values"},
+		Params: []jsonnetAst.Identifier{"releaseName", "namespace", "chartData", "values", "capabilities"},
 		Func: func(args []interface{}) (interface{}, error) {
 			chartData := args[0].([]interface{})
 			releaseName := args[1].(string)
 			namespace := args[2].(string)
 			vals := args[3].(map[string]interface{})
 
+			// If capabilities are set, process them. Use the default
+			// capabilities as the base. If a user provided capabilities, only
+			// copy over the KubeVersion as the other fields are not easily
+			// setable through the jsonnet interface or, in the case of the
+			// helm version, don't make sense.
+			var capabilities = chartutil.DefaultCapabilities.Copy()
+			if len(args) >= 4 {
+				mapCaps := args[4].(map[string]interface{})
+
+				if capKubeVersion, ok := mapCaps["KubeVersion"].(map[string]interface{}); ok {
+					capabilities.KubeVersion = chartutil.KubeVersion{
+						Version: capKubeVersion["Version"].(string),
+						Major:   fmt.Sprintf("%s", capKubeVersion["Major"]),
+						Minor:   fmt.Sprintf("%s", capKubeVersion["Minor"]),
```

Comment:
I know `fmt.Sprintf` is generally a smell, but this seemed like the easiest approach to converting into a string when the value could be `int` or `float`. Happy to change this, but I do feel the performance penalty doesn't warrant doing so.

Reply:
The problem is that `%s` will complain if the value of the interface is not a string (e.g. it will write something like `%!s(int=42)`).

You can either use `%v` or just use `fmt.Sprint(capKubeVersion["Major"])` which will stringify any value

Reply:
I totally meant to do `%v`... sigh. Thank you!